api/services/TagPresenter.js

```diff
@@ -113,15 +118,15 @@ export const fetchAndPresentSummary = (community, tag) =>
     CommunityTag.where({community_id: community.id, tag_id: tag.id})
     .fetch(),
     TagFollow.where({community_id: community.id, tag_id: tag.id})
-    .fetchAll(),
-    Search.forPosts({
-      communities: [community.id],
-      tag: tag.id
-    }).fetchAll(),
+    .count(),
+    bookshelf.knex('posts_tags')
```

Comment:
can you pop this out into a separate method for readability's sake? Maybe make it a class method in CommunityTag?